libs/core/datastructures/include/hpx/datastructures/detail/small_vector.hpp

```diff
@@ -78,22 +78,25 @@ namespace hpx::detail {
             noexcept(Allocator()))
           : memory_()
           , resource_(alloc)
-          , pool_(std::data(memory_), std::size(memory_), &resource_)
+          , pool_(
+                std::data(memory_), std::size(memory_) * sizeof(T), &resource_)
```

Comment:
Oops, good catch! Luckily this is not a memory issue? The missing `sizeof(T)` "only" meant that the actual embedded capacity was `sizeof(T)` smaller, right?

Reply:
Yep, it should have been just that.